<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxSerialPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial Port Config</string>
    </property>
    <widget class="QPushButton" name="pushButtonRefresh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>refresh</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxComName">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelBaud">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>baudrate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>open serial port</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelComName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>serial port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFileOp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QPushButton" name="pushButtonFileLoad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>load file</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFileRun">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>run</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Data</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>121</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditSetCh0"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSetCh1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSetCh2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditGetCh0"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditGetCh1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditGetCh2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>74</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelSetCh0">
        <property name="text">
         <string>channel0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSetCh1">
        <property name="text">
         <string>channel1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSetCh2">
        <property name="text">
         <string>channel2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelGetCh0">
        <property name="text">
         <string>channel0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelGetCh1">
        <property name="text">
         <string>channel1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelGetCh2">
        <property name="text">
         <string>channel2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxCtrl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>serial control</string>
    </property>
    <widget class="QPushButton" name="pushButtonTX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>enable TX</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>enable RX</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxPlot">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>661</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string>plot</string>
    </property>
    <widget class="QWidget" name="widgetPlot" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>641</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
